src/main/resources/ui.properties

```diff
@@ -218,3 +218,14 @@ phoneLabel=Phone
 aliasLabel=Alias
 indexSearchPrompt=Enter a phone number or email address
 assignContactTitle=assign to identities
+next=NEXT
+back=BACK
+finish=FINISH
+nameIdentityInstructions=Please enter the name of the identity, e.g.: "Jane" or "Forentroll"
+nameIdentityPromptText=Name or pseudonym of identity
+emailAddress=E-Mail Address
+emailInstructions=Enter an email address to be found by other Qabel users
```

Comment:
in the Qabel Index Server
(same applies for the phoneNumber which instructions are too short here)